scripts/devenv-builder/configure-vm.sh

```diff
@@ -5,87 +5,118 @@
 #
 set -eo pipefail
 
-BUILD_AND_INSTALL=true
+BUILD_AND_RUN=true
+INSTALL_BUILD_DEPS=true
+FORCE_FIREWALL=false
 RHEL_SUBSCRIPTION=false
 
+start=$(date +%s)
+
 function usage() {
-    echo "Usage: $(basename $0) [--no-build] <openshift-pull-secret-file>"
-    echo "  --no-build   Do not build MicroShift code and install MicroShift RPMs"
+    echo "Usage: $(basename $0) [--no-build] [--no-build-deps] [--force-firewall] <openshift-pull-secret-file>"
+    echo ""
+    echo "  --no-build         Do not build, install and start MicroShift"
+    echo "  --no-build-deps    Do install dependencies for building binaries and RPMs (implies --no-build)"
+    echo "  --force-firewall   Install and configure firewalld regardless of other options"
 
     [ ! -z "$1" ] && echo -e "\nERROR: $1"
     exit 1
 }
 
-# Only RHEL requires a subscription
-if grep -q 'Red Hat Enterprise Linux' /etc/redhat-release ; then
-    RHEL_SUBSCRIPTION=true
-fi
+title() {
+    echo -e "\E[34m$1\E[00m"
+}
 
-if [ $# -ne 1 ] && [ $# -ne 2 ]; then
+while true; do
+    case "$1" in
+    --no-build)
+        BUILD_AND_RUN=false
+        shift
+        ;;
+    --no-build-deps)
+        INSTALL_BUILD_DEPS=false
+        BUILD_AND_RUN=false
+        shift
+        ;;
+    --force-firewall)
+        FORCE_FIREWALL=true
+        shift
+        ;;
+    *) break ;;
```

Comment:
Need to call usage here to avoid invalid options.
```suggestion
    *)  usage ;;
```

Reply:
Done